### PR TITLE
tweets random reminders on Twitter!

### LINKED ISSUE #6: write the part where it sends the selected line to Twitter instead of just printing

### PR DESCRIPTION
**Pull request by ShruAgarwal**
## 🛠️ Fixes Issue

Closes  #6  

## 👨‍💻 Changes proposed

I have made changes to `main.py` file where it selects a random line from `reminders.txt`  and sends it on Twitter. 
I've already tried and tested it so that it tweets after every 4 hours! 

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📄 Note to reviewers

I'm open to feedback for this PR. 😃

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,9 +1,37 @@
...
+
+from time import sleep
 import random
+import tweepy
+
+# Please add your twitter bot KEYS here!
+consumer_key = ""
+consumer_secret = ""
+access_token = ""
+access_token_secret = ""
```
Comment (ykdojo) [inline main.py]: Thank you so much! Any chance you could put this in a separate .env file so it's easier to edit this file?

File: main.py [hunk 2]
```diff
@@ -1,9 +1,37 @@
...
+     # Select a random line from the reminders file.
+     lines = [line[1:] for line in lines if line[0] == "-"]
+     line = lines[random.randint(0, len(lines) - 1)]
+     remind = line.strip()
+    
+     # Random reminder tweets 
+     api.update_status(status=remind)
+     # sleeps for 4 hours
+     sleep(14400)  
+   else:
```
Comment (ykdojo) [inline main.py]: I'm not sure what this else clause is for?

**Review by ykdojo — COMMENTED** [review]
thank you so much!!

let's get this first round of review done :)

File: main.py (hunk 1 above)
Comment (ShruAgarwal) [reply main.py]: I have added a new file `load_env.py` which has a function for loading all the credentials from an `.env` file. 
Make sure to add the *secret keys* in a safe place before hosting the bot. 

**P.S.: If you're planning to host this bot on Heroku, then you can add all the secret keys within Settings > Config Vars**

File: main.py (hunk 2 above)
Comment (ShruAgarwal) [reply main.py]: Sorry, I wrote that by mistake..😅

**Comment by ShruAgarwal** [comment]
Hey @ykdojo 👋
I have made changes in the `main.py` file as suggested. 
Added a new file `load_env.py` for loading all your secret keys! Can't commit .env file cause it's risky! 
Lastly, added `requirements.txt` file for keeping records of all packages.

Let me know your thoughts! 😃

**Comment by ykdojo** [comment]
That's perfect. Thank you so much!!

**Comment by ShruAgarwal** [comment]
Had fun contributing in this project!

Always ready to help 🤗

**Pull request merged**